connect_dataplane_to_lightup.sh

```diff
@@ -7,14 +7,8 @@ if systemctl is-active --quiet lightup-dataplane-connect ; then
     exit 1
 fi
 
-if [ -f init.sh ]; then
```

Comment:
This change breaks the script from being run outside of the repo - see the comment in the else block. E.g.
```
LIGHTUP_TLA=<tla> LIGHTUP_TOKEN=<token> CONNECT_ADMIN_PORT=<880X> CONNECT_APP_PORT=<844X> bash < <(curl -H 'Cache-Control: no-cache' -L https://raw.githubusercontent.com/lupfoss/lupmgr/main/connect_dataplane_to_lightup.sh)
```

Reply:
@jeff-lightup this script is just looking for the ENV vars, right?  Am I missing something else that the connect_dataplane_to_lightup.sh needs from the init.sh?

Reply:
@AbsolutOD more than just the ENV vars - really pretty much everything else too:
- Make sure the proper packages are installed (e.g. need autossh for the lightup-dataplane-connect service) 
- Clones the repo (needed for fixed_config.sh and utils.sh)
- Performs the key setup

Note that on multi node deployments we may run the full `bootstrap.sh` on one machine but on the other machines we'll just run `connect_ssh_to_lightup.sh` and `connect_dataplane_to_lightup.sh`

Reply:
@jeff-lightup Thanks for the context.  I was not aware of those use cases.  Makes sense and I reverse my changes.